parser/src/command.rs

```diff
@@ -242,7 +242,7 @@ fn edit_1() {
 
 #[test]
 fn edit_2() {
-    let input_old = "@bot modify label: +bug.";
+    let input_old = "@bot modify labeled: +bug.";
```

Comment:
Hm, I'm not entirely sure I'm following this test (before or modified), but labeled doesn't look right to me, since I don't think that parses at all right now? We probably want to write a comment here and update the test so that it has the same behavior in terms of what things parse to (and maybe assert that as well).

Reply:
@Mark-Simulacrum It is testing an error there with the assert_ne! below.

https://github.com/rust-lang/triagebot/pull/1591/files#diff-1eb763ec24cd0509bade43f86001d94744947818b6ea7596c4e003c858327395R249

Reply:
There's no guarantee as to what will actually happen in that assert, though? It could just as easily be two different parses or something like that, which seems not great.

Reply:
Agreed. I'll take a look at rewriting a new test